src/components/media/MediaModal.jsx

```diff
@@ -42,7 +42,7 @@ const MediaModal = ({ onClose, onProceed, type, showAltTextModal = false }) => {
 
   const onMediaSelect = (media) => {
     if (methods.watch("selectedMedia")?.name === media.name) {
-      methods.setValue("selectedMedia", {})
+      methods.setValue("selectedMedia", "")
```

Comment:
A little confused here, shouldn't it be an empty object and not an empty string?

Reply:
The check for no item selected here is actually just an existence check, so `{}` mistakenly passes! I've changed this value to `undefined` to more clearly convey the meaning though